packages/data-explorer/src/repository/dataRepository.ts

```diff
@@ -93,7 +93,7 @@ const addFilterIfSet = (request: string, rsqlFilter?: string): string => {
   return rsqlFilter ? `${request}&q=${encodeRsqlValue(rsqlFilter)}` : request
 }
 
-const getTableDataDeepReference = async (tableId: string, metaData: MetaData, coloms: string[], rsqlQuery?: string) => {
+const getTableDataDeepReference = async (tableId: string, metaData: MetaData, coloms: string[], size: Number, rsqlQuery?: string) => {
```

Comment:
Would suggest to make size optional, and maybe use met more descriptive term or add js doc describing the params